demos/3d_segmentation_demo/python/README.md

```diff
@@ -10,6 +10,12 @@ Upon the start-up, the demo reads command-line parameters and loads a network an
 
 ## Preparing to Run
 
+The demo dependencies should be installed before run. That can be achieved with the following command:
+
+```sh
+python3 -m pip install -r <omz_dir>/demos/3d_segmentation_demo/python/requirements.txt
```

Comment:
should it be aligned with other pip instruction format in the repo, e.g. downloader uses
```sh
python3 -mpip install --user -r ...
```

Reply:
changed